docs/modules/user-guide/partials/proc_specifying-persistent-storage.adoc

```diff
@@ -2,7 +2,7 @@
 = Specifying persistent storage
 
 [role="_abstract"]
-Components of any type can specify the custom volumes to be mounted on specific locations within the image. Note that the volume names are shared across all components and therefore this mechanism can also be used to share file systems between components.
+A `container` component can specify the volumes to be mounted on specific locations within the image. If the `container` component mounts the volume, then the volume needs a corresponding volume component of the same name. The volume names are shared across all components, and therefore, this mechanism can also be used to share file systems between components.
```

Comment:
The last sentence here is very wordy. I'm certain this can be shortened while retaining the information. Maybe something like this?

"Volume names are shared across all components. Shared volume names can be used to share file systems between components."